### PR TITLE
.github/pr-check-license.yaml: add license check scan CI

### LINKED ISSUE #3: add github action to do license check
Please refer https://github.com/intel/tdx-tools/blob/main/.github/workflows/pr-check-license.yml

### PR DESCRIPTION
**Pull request by Hulongyin**
This PR is to fix #3 and add badge of `license check`.

### CODE REVIEW

File: .github/workflows/pr-check-license.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: Check License
```
Comment (leyao-daily) [inline .github/workflows/pr-check-license.yaml]: Keep the name in same format with other file

Comment (Hulongyin) [reply .github/workflows/pr-check-license.yaml]: Fixed.

**Review by leyao-daily — APPROVED** [review]
LGTM. Thanks

**Comment by leyao-daily** [comment]
Closes #3

**Pull request merged**